application/src/main/java/com/softserve/lv460/application/security/constants/SecurityConstants.java

```diff
@@ -0,0 +1,29 @@
+package com.softserve.lv460.application.security.constants;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+
+@Data
+@Component
+@PropertySource("classpath:application.properties")
```

Comment:
AFAIK application.properties is a default property source. No need to define it. You can specify in annotation "@ConfigurationProperties(prefix = "jwtconf")" and define fields according to your properties.